src/screens/TokenDetail.js

```diff
@@ -166,15 +187,27 @@ class TokenDetail extends React.Component {
     const renderTokenInfo = () => {
       return (
         <div className="token-general-info">
-          <p className="mb-2"><strong>UID: </strong>{this.state.token.uid}</p>
```

Comment:
Isn't it better to have two components? One for regular tokens and a separate one for NFT tokens? I mean, the more we want to change the NFT screen, the more we have to guarantee we are not messing with the regular token screen.

Reply:
I made it! And i've also broke into three components (one for each section), so we have a NFT version and a non-NFT version that uses these components.